base_multi_company/__manifest__.py

```diff
@@ -1,12 +1,13 @@
-# -*- coding: utf-8 -*-
 # Copyright 2017 LasLabs Inc.
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
 
 {
     'name': 'Multi Company Base',
     'summary': 'Provides a base for adding multi-company support to models.',
     'version': '10.0.1.0.1',
-    'author': "LasLabs, Tecnativa, Odoo Community Association (OCA)",
+    'author': "LasLabs, Tecnativa,"
+              "ACSONE SA/NV,"
```

Comment:
This patch is not enough for granting you co-authorship. Put you in contributors in README instead.

Reply:
Ok no problem. My internal check becomes red due to the coding utf-8 and because I didn't add my company into the module. I'll correct this in few minutes